### DISCUSSION TITLE
请教一下 Safari 下的 Waline 兼容问题。

### DISCUSSION THREAD

**Discussion by first19326** [walinejs/waline#1883] [Q&A]
我知道 Safari 兼容最近 2 年的浏览器，但是我好奇一件事情，就是 Waline 文档的网站是如何在 IOS12 的 Safari 可以正常显示界面的，虽然它依旧不可以加载评论。我想知道这是如何实现的，即使不兼容，能显示界面也是好的，IOS Safari 也没办法调试，所以前来请教。

**Comment by Mister-Hope** [comment]
我没有看懂你表达的意思

**Reply by first19326** [reply]
就是 IOS12 的 Safari 浏览器打开 https://waline.js.org 网站的时候是可以显示评论界面的，但是我项目是 CDN引入的方式，评论界面也没有，所以我就是问下，是怎么处理的。

**Comment by lizheming** [comment]
iOS Safari 可以调试呀 https://zhuanlan.zhihu.com/p/398328458

**Reply by first19326** [reply]
没有 Mac，最初是在使用ios-webkit-debug-proxy尝试了下，发现也没有报错之类的，而且，在windows平台不是特别好用，> ios 13 就不支持了。
